SingularityRunnerBase/src/main/java/com/hubspot/singularity/runner/base/config/MissingConfigException.java

```diff
@@ -0,0 +1,7 @@
+package com.hubspot.singularity.runner.base.config;
+
+public class MissingConfigException extends IllegalStateException {
```

Comment:
not sure `IllegalStateException` is the best exception:

> Signals that a method has been invoked at an illegal or inappropriate time.

Might be best if we just subclass from `RuntimeException`